packages/installer.vm/tools/chocolateyinstall.ps1

```diff
@@ -166,55 +166,13 @@ public class VMBackground
         VM-Write-Log-Exception $_
     }
 
-    # Show dialog that install has been complete
-    Add-Type -AssemblyName System.Windows.Forms
-    Add-Type -AssemblyName System.Drawing
-    # Create form
-    $form = New-Object System.Windows.Forms.Form
-    $form.Text = "$Env:VMname Installation Complete"
-    $form.TopMost = $true
```

Comment:
@Ana06 the reason I went with this version of the dialog window is because it will stay on top of all other Windows (line 175). I think this is better for the user because when they come back to the VM they will see the dialog box before they see anything else.

I'm all for making it smaller, we could even throw it into a function, but I think we need to include some way of making the dialog box sit on top of all the other windows.